saleor/order/utils.py

```diff
@@ -176,15 +177,22 @@ def _calculate_quantity_including_returns(order):
         # count replaced quantity for order
         elif fulfillment.status == FulfillmentStatus.REPLACED:
             quantity_replaced += fulfillment.get_total_quantity()
+        elif fulfillment.status == FulfillmentStatus.WAITING_FOR_APPROVAL:
+            quantity_awaiting_approval += fulfillment.get_total_quantity()
 
     # Subtract the replace quantity as it shouldn't be taken into consideration for
     # calculating the order status
     total_quantity -= quantity_replaced
     quantity_fulfilled -= quantity_replaced
-    return total_quantity, quantity_fulfilled, quantity_returned
+    return (
+        total_quantity,
+        quantity_fulfilled,
+        quantity_returned,
+        quantity_awaiting_approval,
+    )
 
 
-def update_order_status(order: Order):
+def update_order_status(order: Order, confirm: bool = False):
```

Comment:
Maybe we could be more explicit with the name of `confirm` variable? It looks like it is related only to the unconfirmed orders. It could also be misleading, as we use `confrim` word for orders with status != [draft, unconfirmed]. 
Maybe something like `keep_unconfirmed`/`update_unconfirmed`, or something different and more meaningful?